docs/src/pages/company/careers/designer.md

```diff
@@ -88,10 +88,10 @@ If your application stands out, we'll contact you for a get-to-know conversation
 
 If that goes well, you'll be invited to up to four additional conversations, those being:
 
-- A walkthrough on one or two selected projects of your career that you believe demonstrates [comparable experience.](https://articles.uie.com/ux-hiring-lets-talk-about-comparable-experience/)
+- A walkthrough on one or two selected projects of your career that you believe demonstrate [comparable experience.](https://articles.uie.com/ux-hiring-lets-talk-about-comparable-experience/)
```

Comment:
```suggestion
- A walkthrough on one or two selected projects of your career that you believe demonstrates [comparable experience.](https://articles.uie.com/ux-hiring-lets-talk-about-comparable-experience/)
```

At least is what Grammarly is pointing as correct.

Reply:
I get the opposite result from Grammarly. I assume that "demonstrate" is the verb of and "one or two selected" is the noun hence the singular. We need your lights on this one @samuelsycamore.